src/components/Person/Person.jsx

```diff
@@ -1 +1,44 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partnerString = sex === 'm'
+    ? (<p className="Person__partner">{`${partnerName} is my wife`}</p>)
+    : ((<p className="Person__partner">{`${partnerName} is my husband`}</p>)
+    );
```

Comment:
I think this is what we should call DRY, only one word changes, so maybe you can make a pattern, where you put all same things and just change wife or husband. Maybe it's worth to make an additional variable for this